src/main/java/gregtech/common/tileentities/machines/multi/MTEPlasmaForge.java

```diff
@@ -1025,7 +1025,7 @@ private void calculateCatalystIncrease(GTRecipe recipe, FluidStack[] inputFluids
 
         long numberOfOverclocks = (long) (Math.log((double) getMaxInputEu() / recipe.mEUt) / Math.log(4));
         long machineConsumption = recipe.mEUt * (1L << (2 * numberOfOverclocks));
-        double recipeDuration = recipe.mDuration / Math.pow(4, numberOfOverclocks);
+        double recipeDuration = recipe.mDuration / GTUtility.powInt(4, numberOfOverclocks);
```

Comment:
See earlier point on if this would be more optimal as
```suggestion
        double recipeDuration = recipe.mDuration * GTUtility.powInt(4, -numberOfOverclocks);
```

Reply:
See earlier reply.